java-manta-client-kryo-serialization/src/main/java/com/joyent/manta/serialization/EncryptionStateSerializer.java

```diff
@@ -183,7 +183,7 @@ public EncryptionState read(final Kryo kryo, final Input input, final Class<Encr
         final HMac hmac = kryo.readObjectOrNull(input, HMac.class);
 
         final OutputStream cipherStream = EncryptingEntityHelper.makeCipherOutputForStream(
-                multipartStream, encryptionContext, hmac);
+                multipartStream, encryptionContext.getCipherDetails(), encryptionContext.getCipher(), hmac);
```

Comment:
Added a compatibility method and a note to the installation section in the README